diff --git a/build/bazel/remote/asset/v1/BUILD b/build/bazel/remote/asset/v1/BUILD
index aadc6e4..57c1faf 100644
--- build/bazel/remote/asset/v1/BUILD
+++ build/bazel/remote/asset/v1/BUILD
@@ -12,9 +12,8 @@ proto_library(
         "//build/bazel/remote/execution/v2:remote_execution_proto",
         "@com_google_protobuf//:duration_proto",
         "@com_google_protobuf//:timestamp_proto",
-        "@googleapis//:google_api_annotations_proto",
-        "@googleapis//:google_api_http_proto",
-        "@googleapis//:google_rpc_status_proto",
+        "@go_googleapis//google/api:annotations_proto",
+        "@go_googleapis//google/rpc:status_proto",
     ],
 )
 
